recipes/fwdpp/meta.yaml

```diff
@@ -20,14 +20,14 @@ requirements:
     - boost >=1.60.0
 
 build:
-  number: 0
+  number: 1
 package:
   name: fwdpp
-  version: '0.5.3'
+  version: '0.5.4'
 test:
   commands:
-    - fwdppConfig --version | grep 0.5.3 > /dev/null
+    - fwdppConfig --version | grep 0.5.4 > /dev/null
 source:
-  fn:  0.5.3.tar.gz 
-  url: http://github.com/molpopgen/fwdpp/archive/0.5.3.tar.gz
-  md5: 09a00eb8fa1d54c953b5cb4e52a85a1c
+  fn:  0.5.4rc1.tar.gz 
+  url: http://github.com/molpopgen/fwdpp/archive/0.5.4rc1.tar.gz
```

Comment:
Yes, this seem to pass all our tests :)
You see more output if the build actually fails.